chia/data_layer/data_layer.py

```diff
@@ -586,7 +586,8 @@ async def fetch_and_validate(self, store_id: bytes32) -> None:
                 max_generation=singleton_record.generation,
             )
             try:
-                timeout = self.config.get("client_timeout", 15)
+                timeout = self.config.get("client_timeout", 45)
```

Comment:
45 seems low for a total transfer timeout.  But, this pr is increasing from 15, so I won't hold that against it.  Feel free to close this comment for this pr.  Maybe we would prefer a transfer stall timeout or something instead.